src/components/File.jsx

```diff
@@ -67,12 +67,9 @@ class File extends Component {
           <span data-input='checkbox'>
             <input
               type='checkbox'
-              disabled={isDir(attributes)}
               checked={attributes.selected}
              />
-            {isDir(attributes)
-              ? <label />
-              : <label onClick={e => this.toggle(e)} />}
+            <label onClick={e => this.toggle(e)} />
```

Comment:
oh yes,

```jsx
<label onClick={this.toggle.bind(this)} />
```

should do the trick.

But it may be less explicit than

```jsx
<label onClick={e => this.toggle(e)} />
```